docs/site/Understanding-the-differences.md

```diff
@@ -61,28 +61,229 @@ it is independent of the three services. We can define the APIs in facade the
+  </td>
+</tr>
+
+<tr>
+  <td>Boot script</td>
+  <td>Scripts to be invoked during bootstrapping</td>
+  <td><a href="Life-cycle.html">Life cycle events and observers</a></td>
+</tr>
+
+<tr>
+  <td>Remote hooks</td>
+  <td>Before/After hooks for remote methods</td>
+  <td><a href="Interceptors.html">Interceptors</a></td>
+</tr>
+
+<tr>
+  <td>CRUD operation hooks</td>
+  <td>Hooks for CRUD operations</td>
+  <td>(TBD)</td>
+</tr>
+
+<tr>
+  <td>Built-in models</td>
+  <td>Built-in User/AccessToken/Application/ACL/Role/RoleMapping for AAA</td>
+  <td>(TBD)</td>
+</tr>
+
+<tr>
+  <td>Authentication</td>
+  <td>User model as the login provider<br>loopback-component-passport</td>
+  <td><a href="Loopback-component-authentication.html">Authentication component</a>
+    with extensibility to strategy providers
+  </td>
+</tr>
+
+<tr>
+  <td>Authorization</td>
+  <td>Use built-in User/Application/AccessToken model for identity and
+    ACL/Role/RoleMapping for authorization
+  </td>
+  <td><a href="Loopback-component-authorization.html">Authorization component</a>
+    (work in progress).
+  </td>
+</tr>
+
+<tr>
+  <td>Component</td>
+  <td>A very simple implementation to configure and invoke other modules</td>
+  <td>A fully-fledged packaging model that allows contribution of extensions
+    from other modules
+  </td>
+</tr>
+
+<tr>
+  <td>Storage component</td>
+  <td><a href="/doc/en/lb3/Storage-component.html"><code>loopback-compontent-storage</code></a>
+  </td>
+  <td>not available</td>
+</tr>
+
+<tr>
+  <td>Push notifications</td>
+  <td><a href="/doc/en/lb3/Push-notifications.html"><code>loopback-component-push</code></a>
+  </td>
+  <td>not available</td>
+</tr>
+
+<tr>
+  <td>OAuth 2.0 provider</td>
+  <td><a href="/doc/en/lb3/OAuth-2.0.html"><code>loopback-component-oauth2</code></a>
+  </td>
+  <td>(TBD)</td>
+</tr>
+
+<tr>
+  <td>3rd-party logins via Passport</td>
+  <td><a href="/doc/en/lb3/Third-party-login-using-Passport.html"><code>loopback-component-passport</code></a>
+  </td>
+  <td><a href="https://github.com/strongloop/loopback-next/tree/master/extensions/authentication-passport"><code>@loopback/authentication-passport</code></a>
+    <br>(EXPERIMENTAL)
+  </td>
+</tr>
+
+<tr>
+  <td>Offline synchronization</td>
+  <td><a href="/doc/en/lb3/Synchronization.html">Synchronization component</a>
+    (EXPERIMENTAL)
+  </td>
+  <td>not available</td>
+</tr>
+
+<tr>
+  <td>Client SDKs</td>
+  <td>
+    <a href="/doc/en/lb3/AngularJS-JavaScript-SDK.html">AngularJS (1.x)</a>,
+    <a href="/doc/en/lb3/Android-SDK.html">Android</a>,
+    <a href="/doc/en/lb3/iOS-SDK.html">iOS</a>,
+    <a href="/doc/en/lb3/Xamarin-SDK.html">Xamarin</a>,
+    <a href="/doc/en/lb3/LoopBack-in-the-client.html">Isomorphic JavaScript</a>
+  </td>
+  <td>
+    OpenAPI code generators, <br>e.g.
+    <a href="https://swagger.io/tools/swagger-codegen/"><code>swagger-codegen</code></a>
```

Comment:
Could you please list more examples for OpenAPI code generators? Personally I'm still searching for an sdk generator for `Angular 7` which replaces lb3 sdk created by [loopback-sdk-builder](https://github.com/mean-expert-official/loopback-sdk-builder). `swagger-codegen@3` currently does not support `angular`. Migrating the fronend sdk is one of the blockers of our lb4 migration story. This could be helpful for other developers too.

stackoverflow question: [which openapi client sdk for loopback4 and angular?](https://stackoverflow.com/questions/57891183/which-openapi-client-sdk-for-loopback4-and-angular)

Reply:
@derdeka thank you for the suggestion, I'll add `openapi-generator` and `ng-openapi-gen` to the list.

When you find a generator that works well for you, it would be great if you could contribute documentation (or even a short migration guide) to help other LoopBack + Angular users.